### PR TITLE
release: cut the 7.1.0 release

### PR DESCRIPTION
**Pull request by SvetoslavTsenov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<a name="7.1.0"></a>
+# [7.1.0](https://github.com/NativeScript/nativescript-angular/compare/7.0.3...7.1.0) (2018-12-07)
+
+
```
Comment (sis0k0) [inline CHANGELOG.md]:
```suggestion

### Features

* Angular 7.1 support
```

File: e2e/nested-router-tab-view/package.json [hunk 2]
```diff
@@ -30,11 +30,11 @@
     "zone.js": "~0.8.2"
```
Comment (sis0k0) [inline e2e/nested-router-tab-view/package.json]: The dependencies in this file are not updated.

File: ng-sample/package.json [hunk 3]
```diff
@@ -56,7 +56,7 @@
     "nativescript-dev-typescript": "next",
```
Comment (sis0k0) [inline ng-sample/package.json]: The dependencies in this file are not updated.

File: tests/package.json [hunk 4]
```diff
@@ -26,15 +26,15 @@
...
+    "@angular/compiler": "~7.1.0",
+    "@angular/core": "~7.1.0",
+    "@angular/forms": "~7.1.0",
+    "@angular/http": "~7.1.0",
+    "@angular/platform-browser": "~7.1.0",
+    "@angular/platform-browser-dynamic": "~7.1.0",
+    "@angular/router": "~7.1.0",
     "nativescript-angular": "../nativescript-angular",
     "nativescript-unit-test-runner": "^0.3.4",
     "rxjs": "~6.0.0-rc.1",
```
Comment (sis0k0) [inline tests/package.json]:
```suggestion
    "rxjs": "~6.3.3",
```

**Comment by SvetoslavTsenov** [comment]
test

**Pull request merged**